src/key.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { HttpProvider, WSProvider } from '@harmony-js/network'
+import { ChainID } from '@harmony-js/utils'
+import { expect } from 'chai'
+import { HARMONY_RPC_SHARD_0_TESTNET, HARMONY_RPC_SHARD_1_TESTNET, HARMONY_RPC_TESTNET_WS } from './constants'
+import { HarmonyShards } from './interfaces'
+import { Key } from './key'
+
+describe('Key Class', () => {
```

Comment:
You should add a test for `should be an instance of Wallet`

Reply:
Cool, I'll do it right away!